test/runnable/testbounds.d

```diff
@@ -185,34 +185,53 @@ void test13976()
 {
     int[] da = new int[](10);
     int[10] sa;
-    size_t l = 0;               // upperInRange
-    size_t u = 9;               // | lowerLessThan
-                                // | |  check code
-    { auto s = da[l .. u];   }  // 0 0  (u <= 10 && l <= u  )
-    { auto s = da[1 .. u];   }  // 0 0  (u <= 10 && l <= u  )
-    { auto s = da[l .. 10];  }  // 0 0  (u <= 10 && l <= u  )
-    { auto s = da[1 .. u%5]; }  // 0 0  (u <= 10 && l <= u%5)
-
-    { auto s = da[l .. u];   }  // 0 0  (u   <= 10 && l <= u)
-    { auto s = da[0 .. u];   }  // 0 1  (u   <= 10          )
-    { auto s = da[l .. 10];  }  // 0 0  (u   <= 10 && l <= u)
-    { auto s = da[0 .. u%5]; }  // 0 1  (u%5 <= 10          )
-
-    { auto s = sa[l .. u];   }  // 0 0  (u <= 10 && l <= u  )
-    { auto s = sa[1 .. u];   }  // 0 0  (u <= 10 && l <= u  )
-    { auto s = sa[l .. 10];  }  // 1 0  (           l <= u  )
-    { auto s = sa[1 .. u%5]; }  // 1 0  (           l <= u%5)
-
-    { auto s = sa[l .. u];   }  // 0 0  (u <= 10 && l <= u )
-    { auto s = sa[0 .. u];   }  // 0 1  (u <= 10           )
-    { auto s = sa[l .. 10];  }  // 1 0  (           l <= 10)
-    { auto s = sa[0 .. u%5]; }  // 1 1  NULL
+    enum n = 2;
+    enum m = 5;
+    size_t l = 0;                // upperInRange
+    size_t u = 9;                // | lowerLessThan
+                                 // | |  check code
+    { auto s = da[l .. u];     } // 0 0  (u <= 10 && l <= u  )
+    { auto s = da[1 .. u];     } // 0 0  (u <= 10 && l <= u  )
```

Comment:
`1` - not `l` ?

Reply:
I should probably change my font, because all I see is the same character twice in your comment. I assume you mean there's a mixup between number (`1`) and letter (`L`)?

Reply:
Yes, the non-monospace font shows it more clearly:

+    { auto s = da[l .. u];     } // 0 0  (u <= 10 && l <= u  )
+    { auto s = da[1 .. u];     } // 0 0  (u <= 10 && l <= u  )

Reply:
I was trying to figure out why that was even compiling.  Line 190 shows `size_t l = 0; // upperInRange`, so it looks ok, though it could probably use a better name.